shared_model/schema/qry_responses.proto

```diff
@@ -84,6 +84,12 @@ message TransactionsResponse {
   repeated Transaction transactions = 1;
 }
 
+message TransactionsPageResponse {
+  repeated Transaction transactions = 1;
+  string next_tx_hash = 2;
```

Comment:
Should not we use `oneof` here to indicate there are no more transactions following?